src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.VisualStudioTeamServices.cs

```diff
@@ -198,6 +198,56 @@ private static async Task QueueBuildPolicy(GitPullRequest pullRequest, string bu
             }
         }
 
+        /// <summary>
+        /// There is no enum or class in Microsoft.TeamFoundation.SourceControl.WebApi defined for vote values so made my own here.
+        /// Values are documented at https://www.visualstudio.com/en-us/docs/integrate/api/git/pull-requests/reviewers#update-a-reviewers-vote.
```

Comment:
This URL redirects me to the ADO Git REST API docs. https://docs.microsoft.com/en-us/rest/api/azure/devops/git/?view=azure-devops-rest-5.0#update-a-reviewers-vote

Reply:
Updated to https://docs.microsoft.com/en-us/dotnet/api/microsoft.teamfoundation.sourcecontrol.webapi.identityrefwithvote.vote?view=azure-devops-dotnet